openstef/feature_engineering/weather_features.py

```diff
@@ -337,3 +340,93 @@ def add_additional_wind_features(
         )
 
     return data
+
+
+def calculate_dni(radiation: pd.Series, pj: dict) -> pd.Series:
+    """
```

Comment:
Nice docstring, we however use the google style format docstring, https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html . Could you please change the format, most IDE's have a function to autogenerate them.